### DISCUSSION TITLE
Edit Racemenu exported NIF?

### DISCUSSION THREAD

**Discussion by k-basara** [BadDogSkyrim/PyNifly#197] [Q&A]
So I tried editing the head mesh exported from Racemenu (my game is Skyrim AE), export to NIF, then import it with Racemenu but failed.      
Is it because this specific type of NIF file is not supported by this addon, or some extra care/step must be taken for this to be working?    
Thanks.

**Comment by BadDogSkyrim** [comment]
Depends on exactly what you mean by "failed." If your head is lying on its side with some bits warped, that's a known problem with head meshes. The bone transforms they use don't match anything else and I don't know why. You can import the nif, disconnect the armature (which will, I think, revert the head parts to correct positions), and then parent them to an armature you imported separately.

**Reply by k-basara** [reply]
By "failed" I mean head mesh missing from the screen after importing the NIF (which is exported from Blender) in Racemenu (under Sculpt section). I will try again with your method. Thank you.

**Comment by BadDogSkyrim** [comment]
If that doesn't work, send me the mesh you're using and I'll have a look.

**Reply by k-basara** [reply]
Hi. I tried again, with an unedited head exported from the game, unedited in Blender then exported as a new nif, then imported in game, and result is the same, my character's head gone missing.       
I noticed when exporting nif in Blender, there is a warning message from Blender says this: "Some faces have been assigned to more than one partition, which should never happen." The operation completed nonetheless though.     
If I do as you said, reparent the mesh to the skeleton newly imported then export the mesh, the operation will fail with this message: "Export of nif failed, see console window for details".      
How do I send you the files?

**Comment by BadDogSkyrim** [comment]
Can't you attach them here?

I have to update the doc. With the current version, import the nif with "Create bones" turned on. You should get something good in blender. 

But you're trying to export the head again, and use it in game? I didn't realize that was even a thing. I'm not really surprised it doesn't work there's a bone that's named differently in facegen nifs than in the normal skeleton ("Head" rather than "HEAD"). 

The warning about partitions can probably be ignored. The game (sometimes) tolerates overlapping partitions.

**Reply by k-basara** [reply]
Yes it's a throught I recently had, because it's really difficult to make minor tweaks to the head mesh with ingame functionalities.     
I'll attach the nif files here (I'm sorry, I'm not familiar with Github), thank you.
[head-exported.zip](https://github.com/BadDogSkyrim/PyNifly/files/13999769/head-exported.zip)
